xAPI.md

```diff
@@ -313,21 +312,38 @@ an IRL is an IRI that when translated into a URI (per the IRI to URI rules), is
 <a name="def-inverse-functional-identifier" />
 
 __Inverse Functional Identifier__: An identifier which is unique to a particular persona or group.
- Used to identify Agents and Groups.
+
+<a name="def-learning-experience" />
+
+__Learning Experience__: An event associated with the acquisition of knowledge.  It is highly diverse 
```

Comment:
"acquisition of knowledge" is too narrow. (What about skills, abilities and competencies? What about assessment and accreditation of those things? 

"An event associated with learning" would be broader and dodge the issue.